### PR TITLE
Print explicit examples in verbose mode

### LINKED ISSUE #313: Verbose mode doesn't print explicit examples
To reproduce: Add a test with an example decorator, try running it in verbose mode, note that it doesn't print "Trying example:" for the ones provided by the decorators.

### PR DESCRIPTION
**Pull request by Zac-HD**
Closes #313.  Not much else to say when the changelog is longer than the diff.

### CODE REVIEW

**Comment by Zac-HD** [comment]
(note: failure is for changelog because it's already tomorrow in my timezone, but I'll just rerun that once it's tomorrow in UTC too instead of overwriting it twice)

**Comment by Zac-HD** [comment]
Ping @HypothesisWorks/hypothesis-python-contributors for review

**Comment by alexwlchan** [comment]
No tests? 😢

**Comment by Zac-HD** [comment]
Yay tests!  (yep, caught a bug...)

File: scripts/hypothesistooling.py [hunk 1]
```diff
@@ -184,4 +184,4 @@ def modified_files():
 
 def all_files():
     return subprocess.check_output(['git', 'ls-files']).decode(
-        'ascii').split('\n')
+        'ascii').split(os.linesep)
```
Comment (alexwlchan) [inline scripts/hypothesistooling.py]: My instinct would be to reach for `.splitlines()` in this case – is there a difference?

File: tests/cover/test_explicit_examples.py [hunk 2]
```diff
@@ -162,6 +162,20 @@ def test_positive(x):
     assert u'Falsifying example: test_positive(x=-1)' in out
 
 
+def test_prints_verbose_output_for_explicit_examples():
+    @settings(verbosity=Verbosity.verbose)
+    @example('NOT AN INTEGER')
+    @given(integers())
+    def test_positive(x):
```
Comment (alexwlchan) [inline tests/cover/test_explicit_examples.py]: nit: `test_positive` isn’t really an accurate name for this function, is it?

Maybe `test_always_passes`? Or `test_is_not_none`? Just so this isn't inaccurate.

**Comment by Zac-HD** [comment]
Made both suggested changes, thanks 😄

**Comment by DRMacIver** [comment]
All looks good to me on a quick reskim.

As a general observation that is not at all a request for changes, stylistically I would tend to put the space in the string literal rather than the common string as it makes it easier to get layout right, but I don't care what other people do as long as it's correct. :-)

**Pull request merged**